chalk/core.py

```diff
@@ -62,15 +63,17 @@ def compose(
     ) -> Diagram:
         return Compose(envelope, self, other if other is not None else Empty())
 
-    def named(self, name: str) -> Diagram:
+    def named(self, name: Union[str, Tuple[str]]) -> Diagram:
```

Comment:
I think Tuple[str] is only unary tuples. Do you want this be arbitrary tuples?

Reply:
Thanks a lot for the feedback! Yes, I'll correct this and run `mypy` as well (I've postponed running it since I think its latest version (?) shows errors in multiple places in the codebase).